src/lib/components/ApplyModal.svelte

```diff
@@ -41,34 +37,35 @@
   async function createApplication() {
     creatingApplication = true;
 
-    const daiPerDay =
-      (parseInt(total) / parseInt(duration)) * parseInt(durationUnit);
+    const daiPerDay = total / (duration * parseInt(durationUnit));
     const weiPerDay = utils.parseUnits(daiPerDay.toString()).toBigInt();
     const weiPerSecond = weiPerDay / BigInt(86400);
 
-    try {
-      await fetch(
-        `${getConfig().API_URL_BASE}/workstreams/${workstream.id}/applications`,
-        {
-          method: 'POST',
-          credentials: 'include',
-          body: JSON.stringify({
-            letter: applicationText,
-            ratePerSecond:
-              weiPerSecond !== workstream.ratePerSecond.wei
-                ? {
-                    wei: weiPerSecond.toString(),
-                    currency: 'dai'
-                  }
-                : undefined
-          } as ApplicationInput)
-        }
-      );
-    } catch (e) {
-      return;
-    }
-
-    modal.hide();
+    console.log(daiPerDay);
+
+    // try {
+    //   await fetch(
+    //     `${getConfig().API_URL_BASE}/workstreams/${workstream.id}/applications`,
+    //     {
+    //       method: 'POST',
+    //       credentials: 'include',
+    //       body: JSON.stringify({
+    //         letter: applicationText,
+    //         ratePerSecond:
+    //           weiPerSecond !== workstream.ratePerSecond.wei
+    //             ? {
+    //                 wei: weiPerSecond.toString(),
+    //                 currency: 'dai'
+    //               }
+    //             : undefined
+    //       } as ApplicationInput)
+    //     }
+    //   );
+    // } catch (e) {
+    //   return;
+    // }
+
+    // modal.hide();
```

Comment:
Does this not submit it anymore?

Reply:
Sorry, accidentally forgot to push the final commit :D

Reply:
Fixed!

Reply:
Fixed!